Content.Client/UserInterface/Systems/Character/Windows/CharacterWindow.xaml

```diff
@@ -7,6 +7,7 @@
     MinHeight="545">
     <ScrollContainer>
         <BoxContainer Orientation="Vertical">
+            <Label Name="RoleType" VerticalAlignment="Top" Margin="0 6 0 10" HorizontalAlignment="Center" StyleClasses="LabelHeading" Access="Public"/>
```

Comment:
I'm not sure I like the RoleType being centered at the top of the character window.

I think I'd prefer it to be below the character information, and maybe in a different PR we can add role descriptions for where the "Antag Roles" placeholder is.

Reply:
I think the way to deal with the anag roles placeholder is to give all antags Objectives, even if they are just "dummies" that cannot actually be "completed", to guide their play. All Team Antagonists, for example, should have an objective that specifies or at least indicates what their team is